src/libs/deprecatedAPI.js

```diff
@@ -330,19 +330,6 @@ function Report_GetHistory(parameters) {
     return Network.post(commandName, parameters);
 }
 
-/**
- * @param {Object} parameters
- * @param {Number} parameters.reportID
- * @param {Boolean} parameters.pinnedValue
- * @returns {Promise}
- */
-function Report_TogglePinned(parameters) {
```

Comment:
I noticed that we refactored this command but didn't remove it from deprecatedAPI. So I'm removing it here